src/views/layouts/login.hbs

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Lato|Montserrat" rel="stylesheet">
+    <link rel="stylesheet" href="/css/login.css">
```

Comment:
Your signup page is supposed to be using this layout too, so it would be better if you passed the CSS files and JS files dynamically. i.e. say `res.render('login', { style: 'login.css', script: 'login.js' });` and in your hbs `<link rel="stylesheet" href="/css/{{style}}">`